server/auth/types/authentication_type.test.ts

```diff
@@ -106,4 +158,47 @@ describe('test tenant header', () => {
     const result = await dummyAuthType.authHandler(request, response, toolkit);
     expect(result.requestHeaders.securitytenant).toEqual('dummy_tenant');
   });
+
+  it(`keepalive should not shorten the cookie expiry`, async () => {
```

Comment:
rename it to: `keepalive cookie expire can be overridden with a greater value` (or any value?)

Reply:
I think the name is fine - i had some work that i forgot to push up so maybe it was confusing. I am testing that keepalive does not shorten the existing expiry of the cookie.